packages/react-components/src/components/styles/StreamMedia.styles.ts

```diff
@@ -21,7 +21,18 @@ export const mediaContainer = (theme: Theme): string =>
 /**
  * @private
  */
-export const invertedVideoStyle = (theme: Theme): string =>
+export const invertedVideoInPipStyle = (theme: Theme): string =>
   mergeStyles(mediaContainer(theme), {
-    transform: 'rotateY(180deg)'
+    // The HTMLElement returned by the headless SDK is already inverted.
```

Comment:
This relies on the headless SDK not changing their implementation.. could we have the calling SDK fix this on their render element instead? (I can't see a way with just css though)

Reply:
We should probably file a bug on them regardless as everyone using their render element will hit this bug

Reply:
I'm already talking to the headless folks. This PR is a stop-gap.